pom.xml

```diff
@@ -130,7 +130,7 @@
         <gravitee-notifier-slack.version>1.3.0</gravitee-notifier-slack.version>
         <gravitee-alert-api.version>1.8.0</gravitee-alert-api.version>
         <gravitee-notifier-api.version>1.4.1</gravitee-notifier-api.version>
-        <gravitee-risk-assessment-api.version>1.0.1</gravitee-risk-assessment-api.version>
+        <gravitee-risk-assessment-api.version>2.0.0-alpha.1</gravitee-risk-assessment-api.version>
```

Comment:
What is the purpose of this update in this PR ?

Reply:
We forgot about it updating it,

We can make it in a separate PR if you wish

Reply:
That's fine here, just want to be sure that was not a mistake 😊